tests/e2e/specs/test.js

```diff
@@ -1,8 +1,22 @@
 // https://docs.cypress.io/api/introduction/api.html
 
-describe('My First Test', () => {
-  it('Visits the app root url', () => {
-    cy.visit('/')
-    cy.contains('h1', 'Welcome to Your Vue.js App')
+describe('Mesh landing page after login', () => {
+  it('Visits the app root url and renders without errors', () => {
+    cy.visit('/localhost:8080')
+    cy.contains('h3', 'Hi, Nikki')
+    cy.get('.left-section').find('a').should('have.length', 4)
+    cy.get('.projects-container').find('a').should('have.length', 4)
+
+    cy.get('.feedback-container').should('exist')
+    cy.get('.notes-container').should('exist')
+    cy.contains('footer', '🦑')
+  })
+})
+
+// this one isn't working for some reason. I think I need to change the route or test the route by clicking???
+describe('Mesh about page', () => {
+  it.skip('Visits the about route and renders without errors', () => {
```

Comment:
This doesn't run on my local when I unskip it. Should it?

Reply:
it's not really getting access to the about page, I think because it's a dynamically rendered route and you have to actually click on something to load it. I have to think about how to deal with that. But it shouldn't prevent getting this pushed up for now.